core/sql/exp/exp_conv.cpp

```diff
@@ -1445,6 +1445,10 @@ ex_expr::exp_return_type convUnicodeToDatetime(char *target,
    }
 
    ExRaiseSqlError(heap, diagsArea, EXE_CONVERT_DATETIME_ERROR);
+   char hexstr[256];
```

Comment:
Also what do you think about making this length longer than 256? We are dealing with strings that are much longer quote often. I think ComCondition is quite flexible, though I don't know what the max length of stringParams are. Since this is the error case, we don't need to be too concerned about performance (though for loads we may continue after error). Some balance will have to be struck. Maybe this should be configurable with a session setting or env var? I am happy with 256 too, if you think that is sufficient for now.